app/javascript/pages/howToApply/how-to-apply.tsx

```diff
@@ -1,14 +1,28 @@
-import React, { ReactNode } from "react"
-
+import React, { ReactNode, useEffect, useState, useContext } from "react"
```

Comment:
This file is now over the recommended max length of 250 lines of code. I'll ignore this issue in this PR, but I'm going to make a follow-up ticket to refactor by moving most of the components into separate files in `app/javascript/modules/howToApply`.